docs/pages/jpscore/jpscore_demos.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Demos
+keywords: simulation
+tags: [jpscore, test]
+sidebar: jupedsim_sidebar
+folder: jpscore
+permalink: jpscore_demos.html
+summary: Introduction of demos in jpscore.
+
+last_updated: Feb 04, 2019
+---
+
+## 01_corridor
+### Description
+This demo shows 20 agents moving from left to right in a rectangular corridor.
+
+The length of the corridor is 10 meters and the width is 2 meters.
+
+### Settings
+- Model: [Generalized Centrifugal Force Model](http://www.jupedsim.org/jpscore_operativ.html#generalized-centrifugal-force-model)
```

Comment:
Please don't use these links. 

You can link to internal files by just using the names of the files.

For example 

```
[Generalized Centrifugal Force Model](jpscore_operativ.html#generalized-centrifugal-force-model)
```